Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -225,10 +225,25 @@ public override string ToString()
 
     public class Wheel
     {
-        [IonConstructor]
-        public Wheel([IonPropertyName("specification")] string specification)
+        [IonField]
+        internal string specification;
+
+        [IonField]
+        internal int size;
+
+        public Wheel(int size)
```

Comment:
This *can* be private and we should have an explicit test for that

Reply:
Do you mean the `[IonConstructor]` tagged constructor can be private and we should test that?